src/test/java/tictactoe/BoardTest.java

```diff
@@ -162,4 +163,16 @@ void returnsAvailableSquaresOnTheGrid() {
 
         assertEquals(expectedSquares, availableSquares);
     }
+
+    @Test
+    void returnsANewInstanceOfBoard() {
+        var squares = Arrays.asList("x", "o", "x", "4", "o", "o", "7", "x", "x");
+        var grid = new Grid(squares);
+        var board = new Board(grid);
+
+        var copyBoard = board.copyBoard();
+
+        assertNotEquals(board, copyBoard);
+        assertThat(copyBoard).isInstanceOf(Board.class);
```

Comment:
Think this test should be confirming that it has the same squares even if they're not equal.

Reply:
Good point, I'll add this in. Thanks!